### PR TITLE
Added note and updated text

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/allow-or-prevent-custom-script.md [hunk 1]
```diff
@@ -95,7 +95,9 @@ To allow custom script on a particular site (previously called "site collection"
     Set-PnPSite <SiteURL> -NoScriptSite $false
     ``` 
 
-If you change this setting for a user's OneDrive or a classic team site, it will be overridden by the Custom Script setting in the admin center within 24 hours.
+If you change this setting for a classic team site, it will be overridden by the Custom Script setting in the admin center within 24 hours.
+> [!NOTE]
+    > Allow/disallow of custom script to individual OneDrive’s cannot be done.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/allow-or-prevent-custom-script.md:100]:
```suggestion
> You cannot allow or prevent custom scripts to an individual user's OneDrive.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PramodBalusu, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**